packages/story-editor/src/elements/video/display.js

```diff
@@ -112,7 +112,7 @@ function VideoDisplay({ previewMode, box: { width, height }, element }) {
           poster={poster || resource.poster}
           style={style}
           {...videoProps}
-          preload="metadata"
+          preload="all"
```

Comment:
preload all is also used in elements > gif > display, and the equivalent (true) is ued in components > helpCenter > quickTip > index. 

it might be worth consolidating these for the future to avoid thinking they do something different.